desktop/src/main/java/bisq/desktop/main/dao/governance/proposals/ProposalsView.java

```diff
@@ -203,6 +209,8 @@ protected void activate() {
 
         sortedList.comparatorProperty().bind(tableView.comparatorProperty());
 
+        maybeShowDAOTestingFeedbackWindow();
```

Comment:
Seems that would show the popup after the user has made a proposal. We want to show the popup only after the voting phase in case the user has created a proposal or if he has voted. Sorry that I have not been more clear in the GH issue....